internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -150,6 +150,11 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				placeholderReplacer := strings.NewReplacer("{account_id}", accountID, "{zone_id}", zoneID)
 				endpoint = placeholderReplacer.Replace(endpoint)
 
+				// If the v4 API client has not been initialized, then initialize it.
+				if api == nil {
```

Comment:
the client is already initialised at https://github.com/cloudflare/cf-terraforming/blob/master/internal/app/cf-terraforming/cmd/root.go#L16 so this isn't required.